### PR TITLE
Call crossgenerate using CLI

### PR DESCRIPTION
**Pull request by marieke-ebar**
Please review before merging this into main. This merger seems like the best option to me in terms of commit history

### CODE REVIEW

File: src/tryout.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+function test(): void {
```
[the remark this answers is not in the document]
Comment (marieke-ebar) [reply src/tryout.ts:1]: Yes, I forgot to remove it

File: package.json [hunk 2]
```diff
@@ -15,10 +15,17 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "jsonpath": "^1.1.1",
```
[the remark this answers is not in the document]
Comment (marieke-ebar) [reply package.json:18]: Correct

File: package.json [hunk 3]
```diff
@@ -15,10 +15,17 @@
...
+    "jsonpath-plus": "^7.2.0",
     "mustache": "^4.2.0"
   },
   "devDependencies": {
-    "@types/node": "^18.11.18",
+    "@types/jsonpath": "^0.2.0",
```
[the remark this answers is not in the document]
Comment (marieke-ebar) [reply package.json:23]: Correct as well, I overlooked that, thanks

**Comment by marieke-ebar** [comment]
I updated the files in response to the feedback, I'd say this can be merged now

**Review by harmen-xb — APPROVED** [review]

**Pull request merged**